cob_bringup/drivers/usb_camera_node.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<launch>
+  <arg name="camera_name" default="head_cam"/>
```

Comment:
I would remove the default entry and thus make it a required parameter as for the other launch files in cob_driver too.